test/functional/tests.py

```diff
@@ -577,31 +577,31 @@ def test_dir_make_fill_list_delete(self):
             self.api.files_stat(self.test_directory_path)
 
 
-@skipIfOffline()
-class TestBlockFunctions(unittest.TestCase):
-    def setUp(self):
-        self.api = ipfsapi.Client()
-        self.multihash = 'QmYA2fn8cMbVWo4v95RwcwJVyQsNtnEwHerfWR8UNtEwoE'
-        self.content_size = 248
-
-    def test_block_stat(self):
-        expected_keys = ['Key', 'Size']
-        res = self.api.block_stat(self.multihash)
-        for key in expected_keys:
-            self.assertTrue(key in res)
-
-    def test_block_get(self):
-        self.assertEqual(len(self.api.block_get(self.multihash)), self.content_size)
-
-    def test_block_put(self):
-        path = os.path.join(os.path.dirname(os.path.dirname(__file__)),
-                            "functional", "fake_dir", "fsdfgh")
-        expected_block_multihash = 'QmPevo2B1pwvDyuZyJbWVfhwkaGPee3f1kX36wFmqx1yna'
-        expected_keys = ['Key', 'Size']
-        res = self.api.block_put(path)
-        for key in expected_keys:
-            self.assertTrue(key in res)
-        self.assertEqual(res['Key'], expected_block_multihash)
+# @skipIfOffline()
+# class TestBlockFunctions(unittest.TestCase):
+    # def setUp(self):
+        # self.api = ipfsapi.Client()
+        # self.multihash = 'QmYA2fn8cMbVWo4v95RwcwJVyQsNtnEwHerfWR8UNtEwoE'
+        # self.content_size = 248
+
+    # def test_block_stat(self):
+        # expected_keys = ['Key', 'Size']
+        # res = self.api.block_stat(self.multihash)
+        # for key in expected_keys:
+            # self.assertTrue(key in res)
+
+    # def test_block_get(self):
+        # self.assertEqual(len(self.api.block_get(self.multihash)), self.content_size)
+
+    # def test_block_put(self):
+        # path = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                            # "functional", "fake_dir", "fsdfgh")
+        # expected_block_multihash = 'QmPevo2B1pwvDyuZyJbWVfhwkaGPee3f1kX36wFmqx1yna'
+        # expected_keys = ['Key', 'Size']
+        # res = self.api.block_put(path)
+        # for key in expected_keys:
+            # self.assertTrue(key in res)
+        # self.assertEqual(res['Key'], expected_block_multihash)
```

Comment:
Why comment out the block test?

Reply:
I commented that out when I was testing on my machine. For some reason it was taking a really long time and I forgot to uncomment it. My bad.